docs/docs/50-roadmap.md

```diff
@@ -17,20 +17,26 @@ Project](https://github.com/akuity/kargo/milestones)
 
 ## In Progress
 
-### v0.7.0
+### v0.8.0
 
-__Status:__ In Progress
-__Expected:__ 2024-06-07
+__Expected:__ 2024-07-12
 
 | Name | Type | Description |
 | ---- | ---- | ----------- |
-| Multiple `Warehouse`s | feature | Improve UI support for multiple Freightlines rooted in different `Warehouse`s. |
-| Manual `Freight` Creation | feature | Add CLI and UI support for manual `Freight` creation. This will give users the flexibility to create novel combinations of artifacts that `Warehouse`s will not -- for instance, pairing the most recent version of a container image with an _older_ version of application manifests. |
-| ECR/GitHub Auth | feature | Native authentication support for ECR registries and using GitHub applications | 
-| [Patch Promotions](https://github.com/akuity/kargo/issues/1250) | poc | Support a generalized option to promote arbitrary configuration (e.g. strings, files, and directories) to other paths of a GitOps repository. |
-| Production Readiness | chore | <ul><li>Prioritize stability of existing features.</li><li>Pay down technical debt.</li><li>**This is not a guarantee that v0.7.0 will be production-ready. It is a commitment to large steps in that direction.**</li></ul> |
+| Multiple `Freight` per `Stage` | feature | The overarching theme of the release: Permit `Stage`s to host multiple pieces of `Freight` from different `Warehouse`s. This will permit different artifacts, or sets of artifacts, to be promoted through a pipeline with different/independent cadence. |
+| Production Readiness | chore | <ul><li>Prioritize stability of existing features.</li><li>Pay down technical debt.</li><li>**This is not a guarantee that v0.8.0 will be production-ready. It is a commitment to large steps in that direction.**</li></ul> |
+
+## Upcoming
+
+### v0.9.0
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| Auth via [GitHub Apps](https://docs.github.com/en/apps) | feature | Support GitHub Apps as an authentication option for GitHub and ghcr.io repositories. | 
```

Comment:
This might moved back to v0.8.0 depending on what @jessesuen learns about its urgency or lack thereof.

Reply:
It has moved back to v0.8.0.